rust-runtime/aws-smithy-checksums/src/body.rs

```diff
@@ -0,0 +1,444 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+use crate::http::{new_from_algorithm, HttpChecksum};
+
+use aws_smithy_http::body::SdkBody;
+use aws_smithy_http::header::append_merge_header_maps;
+use aws_smithy_types::base64;
+
+use bytes::Bytes;
+use http::header::HeaderName;
+use http::{HeaderMap, HeaderValue};
+use http_body::{Body, SizeHint};
+use pin_project_lite::pin_project;
+
+use std::fmt::Display;
+use std::pin::Pin;
+use std::task::{Context, Poll};
+
+// pin_project_lite can't handle triple-slash doc comments
```

Comment:
Wait, but `///` doc comments are used below?

Reply:
Another leftover from when I thought I needed to use doc attributes for docs in `pin_project` blocks, thanks for catching.